salt/state.py

```diff
@@ -1642,7 +1644,7 @@ def requisite_in(self, high):
         errors.extend(req_in_errors)
         return req_in_high, errors
 
-    def call(self, low, chunks=None, running=None):
+    def call(self, low, chunks=None, running=None, retries=1):
```

Comment:
As I read the code below, this looks like two retries will happen by default unless specifically set not to. I don't think we should be retrying unless the user explicitly requests it. Is my understanding here correct?